pandas/io/json/arrow_json_parser_wrapper.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pandas._typing import ReadBuffer
+from pandas.compat._optional import import_optional_dependency
+
+from pandas.core.dtypes.inference import is_integer
+
+if TYPE_CHECKING:
+    from pandas import DataFrame
+
+
+class ArrowJsonParserWrapper:
+    """
+    Wrapper for the pyarrow engine for read_json()
+    """
+
+    def __init__(self, src: ReadBuffer[bytes]) -> None:
+        super().__init__()
+        self.src = src
+
+    def _parse_kwd(self) -> None:
+        """
+        Validates keywords before passing to pyarrow
+        """
+        ...
```

Comment:
Might be nice to declare these methods as `@abstractmethod`

Reply:
Yeah we can also remove it. For the arrow json parser, I got the logic from the read_csv arrow parser. Kept these methods as a placeholder just in case but they are not used right now.